client/lib/login/index.js

```diff
@@ -121,14 +122,6 @@ export function getSignupUrl( currentQuery, currentRoute, oauth2Client, locale,
 		return `${ signupUrl }/wpcc?${ oauth2Params.toString() }`;
 	}
 
-	if ( oauth2Client && isJetpackCloudOAuth2Client( oauth2Client ) ) {
```

Comment:
I'm curious why these codes are removed.

Reply:
It was basically doing the same thing: 

<img width="539" alt="Screenshot 2024-02-15 at 3 05 57 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/5d24a2a8-857b-4553-b90d-e02097747a69">

Reply:
Ahh thanks for the clarifications.